accurics-aws-tf-sample.tf

```diff
@@ -231,23 +245,104 @@ resource "aws_dynamodb_table" "accurics-test-dynamodbtable1" {
     type = "S"
   }
   server_side_encryption {
-    enabled     = false
+    enabled = false
   }
   tags = {
-    Name = format("%s-dynamodbtable1", var.acqaPrefix)
+    Name         = format("%s-dynamodbtable1", var.acqaPrefix)
     ACQAResource = "true"
-    Owner = "Accurics"
+    Owner        = "Accurics"
   }
+  point_in_time_recovery = "{\n    \"point_in_time_recovery\": {\n        \"enabled\": true\n    }\n}"
 }
 
 # Codecommit
 resource "aws_codecommit_repository" "accurics-test-ccrepo1" {
   repository_name = "accurics-test-ccrepo1"
   description     = "accurics-test-ccrepo1"
   tags = {
-    Name = format("%s-cloudfront1", var.acqaPrefix)
+    Name         = format("%s-cloudfront1", var.acqaPrefix)
     ACQAResource = "true"
-    Owner = "Accurics"
+    Owner        = "Accurics"
+  }
+}
+
+
+resource "aws_s3_bucket_policy" "accurics-test-s3bucket1Policy" {
+  bucket = "${aws_s3_bucket.accurics-test-s3bucket1.id}"
+
+  policy = <<POLICY
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Sid": "accurics-test-s3bucket1-restrict-access-to-users-or-roles",
+      "Effect": "Allow",
+      "Principal": [
+        {
+          "AWS": [
+            "arn:aws:iam::##acount_id##:role/##role_name##",
+            "arn:aws:iam::##acount_id##:user/##user_name##"
+          ]
+        }
+      ],
+      "Action": "s3:GetObject",
+      "Resource": "arn:aws:s3:::${aws_s3_bucket.accurics-test-s3bucket1.id}/*"
+    }
+  ]
+}
+POLICY
+}
+resource "aws_flow_log" "accurics-test-vpc1" {
+  vpc_id          = "${aws_vpc.accurics-test-vpc1.id}"
+  iam_role_arn    = "##arn:aws:iam::111111111111:role/sample_role##"
+  log_destination = "${aws_s3_bucket.accurics-test-vpc1.arn}"
```

Comment:
**Accurics remediation**
 
**[MEDIUM]: DynamoDB read-write capacities must be reserved**
 
**Vulnerability**
DynamoDB's Capacity is NOT Reserved for table reads and writes. When you create a new provisioned table in Amazon DynamoDB, you must specify its provisioned throughput capacity. This is the amount of read and write activity that the table can support. DynamoDB uses this information to reserve sufficient system resources to meet your throughput requirements.
 
**Remediation**
Make sure to change billing mode to PROVISIONED. This is also the default value in terraform. By changing this mode to PROVISIONED, it'll become mandatory to provide read read and write capacity, that will eventually reserve system resources.

source: accurics-app